README.md

```diff
@@ -33,8 +34,64 @@ aimed at react developers that want to embed the widgets into their applications
 npm install --save webex @webex/widgets
 ```
 
+## Webex Components vs Webex Widgets
+
+[Webex Components](https://github.com/webex/components#webex-components) is a set of React components that _only_ follow webex standard styling without any data being injected to them.
```

Comment:
I don't like that you say "only ... styling" because it makes it sounds like they're only presentational components. We know they're more than that. While data is not included with the components, they understand it, and that's a big difference.

```suggestion
[Webex Components](https://github.com/webex/components#webex-components) is a set of React components that follow webex standard styling to which data is injected to them via adapters.
```